packages/syft/src/syft/service/worker/worker_image_service.py

```diff
@@ -108,16 +111,15 @@ def build(
     )
     def get_all(
         self, context: AuthedServiceContext
-    ) -> Union[List[SyftWorkerImage], SyftError]:
+    ) -> Union[DictTuple[str, SyftWorkerImage], SyftError]:
         """
         One image one docker file for now
-        TODO: change repr when listing
         """
         result = self.stash.get_all(credentials=context.credentials)
         if result.is_err():
             return SyftError(message=f"{result.err()}")
-
-        return result.ok()
+        images: List[SyftWorkerImage] = result.ok()
+        return DictTuple((im.full_tag, im) for im in images)
```

Comment:
Q: What is the functionality of DictTuple, even though , we have specific columns, I think __repr_attrs__ overrides as the full_tag field is not displayed.


<img width="963" alt="Screenshot 2023-12-22 at 10 44 56 AM" src="https://github.com/OpenMined/PySyft/assets/43314053/f6f0ca94-d2a7-43c6-9ac0-e98c64ff9a22">